src/org/typo3/chefci/v2/stages/Publish.groovy

```diff
@@ -21,6 +21,8 @@ class Publish extends AbstractStage {
      */
     protected publish() {
 
+        (new Slack(script)).notifyVersionBump()
```

Comment:
you could think about injecting slack as an object, when a stage is added to a pipeline - like this you can get rid of the `new ...` and just call `slack.notifyVersionBump()`

You can put the `slack` property and the `setSlack(...)` inject method into the abstract class, so you don't blow up the implementations...